OpenSim/Sandbox/MatlabScripts/timesSeriesTable.m

```diff
@@ -0,0 +1,195 @@
+function output_table = timesSeriesTable(generictable)
+%% // Convert between OpenSim and Matlab Timeseries 
+%  
+%   Inputs can be either an OpenSim TimeSeriesTable, TimesSeriesTableVec3,
+%   or a Matlab times series collection. 
+%
+%   Outputs from OpenSim TimesSeriesTable's will be a Matlab times series
+%   collection. 
```

Comment:
We should maybe give a link to MATLAB documentation about `tscollection` and `timeseries`, and maybe also have some examples of how to use it.